src/AppInstallerCLICore/Workflows/ConfigurationFlow.h

```diff
@@ -5,6 +5,12 @@
 
 namespace AppInstaller::CLI::Workflow
 {
+    // Fails if the configuration file path does not exist.
+    // Required Args: ConfigurationFile
+    // Inputs: None
+    // Outputs: None
+    void EnsureConfigurationFileExists(Execution::Context& context);

```

Comment:
This name of this almost makes me think that this could be abstracted into a generic check for files existing where an enum could provide the logic of which arg to grab the path from . `winget import` is one that comes to mind, install/uninstall/show with `--manifest`, `winget hash`, etc. I'm surprised this isn’t genericised already. . .

Reply:
It actually is, and I wrote it. I just didn't look hard enough to rediscover `VerifyFile`.